extras/gedit/conky.lang

```diff
@@ -0,0 +1,716 @@
+        <context ref="template"/>
+        <context ref="date"/>
+	<context ref="number"/>
+        <context ref="keywordsConfig"/>
+	<context ref="keywordsText"/>
+	<context ref="boolean"/>
+        <context ref="predefinedColors"/>
+	<context ref="windowType"/>
+	<context ref="windowHints"/>
+	<context ref="alignment"/>
+	<context ref="argumentsText"/>
+	<context ref="argumentsConfig"/>
+      </include>
+    </context>
+
+<!-- ======================================================================================= -->
+<!-- Keep in mind gedit does not like < and >, use &lt; and &gt; instead! -->
+
+    <!-- Block comment, lua syntax -->
+    <context id="block-comment" style-ref="comment" class="comment" class-disabled="no-spell-check">
+      <start>--\[(=*)\[</start>
+      <end>]\%{1@start}]</end>
+      <include>
+	<context ref="def:in-comment"/>
+      </include>
+    </context>
+
+    <!-- Line comment, lua syntax -->
+    <!-- Note: this will only highlight line comments BEFORE the conky.text section -->
+    <context id="line-comment" style-ref="comment" class-disabled="no-spell-check">
+      <match>(?&lt;!conky.text)(\-\-.*)</match>
+    </context>
+
+    <!-- File paths and URLs, starting with https://, http://, ~/ or / -->
+    <context id="path" style-ref="path" class="string">
+      <match>((?:\~\/|\/|htt(?:p|ps)\:\/\/){1}[^\s\}\']+)</match>
+    </context>
+
+    <!-- Brackets --> 
+    <!-- BUG: this will also highlight the last bracket from the conky.config section -->
+    <context id="brackets" style-ref="brackets">
+      <match>(\$\{|(?&lt;!\\)\})</match>
+    </context>
+
+    <!-- Custom colors -->
+    <context id="colors" style-ref="argument">
+      <match>\'([\da-fA-F]{6})\'</match>
+    </context>
+
+    <!-- Fonts -->
+    <!-- Note: abusing | , it's a trick to unmatch :) http://www.rexegg.com/regex-best-trick.html#notarzan -->
+    <!-- BUG: this will not only trigger for font -->
+    <context id="fonts" style-ref="argument">
+      <match>(?&lt;=font)|(\b(?:[\w_-]\s?)*(\:(style\=)?([Mm]edium)?([Bb]old|[Ii]talic))?\:(?:pixel)?size=[0-9]+)</match>
+    </context>
+
+    <!-- Date arguments -->
+    <!-- BUG: will also trigger with keywords other than time -->
+    <context id="date" style-ref="argument">
+      <match>\%[aAbBcCdDeDgGhHIjmMnprRStTuUVwWxXyYzZ%]{1}(?=(?:(?!\$\{).)*\})</match>
+    </context>
+
+    <!-- Template arguments -->
+    <!-- BUG: will also trigger between [[ ]] other than from template\d{1} -->
+    <context id="template" style-ref="argument">
+      <match>\\[0-9]{1}(?=(?:(?!\[\[).)*\]\])</match>
+    </context>
+
+    <!-- Numbers -->
+    <context id="number" style-ref="decimal">
+      <match>(?&lt;=[\ \=]|\dx)([\+\-]{0,1}\d+)([\.\,]{1}\d+)?</match>
+    </context>
+
+<!-- ======================================================================================= -->
+<!-- The keywords down below are actually regexes too, order is important! foobar must come before foo -->
```

Comment:
Is this comment correct? A number of keywords in the `keywordsText` section don't follow this rule but still function correctly in gedit 3.18.

e.g.

- `addr`
- `apcupsd`
- `apcupsd_load`
- `audacious_length`
- `battery`
- `cpu`

to name a few